runtime/onert/core/src/compiler/ShapeValidator.cc

```diff
@@ -109,18 +109,6 @@ void ShapeValidator::visit(const ir::operation::BatchToSpaceND &node)
   OP_REQUIRES(input_shape.C == output_shape.C);
 }
 
-void ShapeValidator::visit(const ir::operation::Comparison &node)
-{
-  const auto output_index{node.getOutputs().at(0)};
```

Comment:
Since comparison may needs some shape validation (e.g., lhs.shape == rhsl.shape)
How about adding `// TODO write shape validation of comparison`?

Reply:
OK. I'll add comment